Assets/MixedRealityToolkit-SDK/Features/Utilities/Solvers/SurfaceMagnetism.cs

```diff
@@ -517,10 +570,9 @@ private void FindPlacementPlane(Vector3 origin, Vector3 direction, Vector3[] pos
             }
 
             // Figure out how far the plane should be.
-            if (!useClosestDistance && closestPoint >= 0.0f)
+            if (!useClosestDistance && closestPoint >= 0)
             {
                 float centerPlaneDistance;
-
```

Comment:
seems like a rather unnecessary formatting change

Reply:
I don't consider mrtk_development to be the source truth for solvers, as they don't work in their current state.

Reply:
Right, and changing these lines isn't required to get it to work.

Reply:
This makes the code more readable, and since we're in the middle of a port, it seems like the perfect time to polish up. I'm usually pretty against unnecessary code churn, but I don't think this should be reverted.

Reply:
Good points from both of you. I unreverted these in my last commit.